command.go

```diff
@@ -685,11 +685,22 @@ func (cmd *ZSliceCmd) readReply(cn *pool.Conn) error {
 
 //------------------------------------------------------------------------------
 
+// Scanner is a ScanCmd with Iterator capabilities
+type Scanner struct {
+	*ScanCmd
+	c *commandable
+}
```

Comment:
Why not to remove `Scanner` and add `Iterator() *ScanIterator` method to `ScanCmd`?